code/game/objects/items/inducer.dm

```diff
@@ -131,13 +144,36 @@
 		return TRUE
 	recharging = FALSE
 
+/obj/item/inducer/proc/do_harm(mob/living/carbon/victim, mob/living/user)	//maybe a good idea to leave this out?
```

Comment:
Is this even called from anywhere? If it's just commented out code, might as well remove it

Reply:
I was considering removing it altogether, but changed my mind.